### PR TITLE
Update strings.xml

### PR DESCRIPTION
**Pull request by logaprakash**
Little change in the grammar.

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -302,13 +302,13 @@
...
 
     <string name="forbidden_permissions">You do not have permission %s</string>
-    <string name="forbidden_permissions_rename">to rename this file</string>
-    <string name="forbidden_permissions_delete">to delete this file</string>
-    <string name="share_link_forbidden_permissions">to share this file</string>
-    <string name="unshare_link_forbidden_permissions">to unshare this file</string>
-    <string name="update_link_forbidden_permissions">to update this share</string>
-    <string name="forbidden_permissions_create">to create the file</string>
-    <string name="uploader_upload_forbidden_permissions">to upload in this folder</string>
+    <string name="forbidden_permissions_rename">To rename this file</string>
```
Comment (jancborchardt) [inline res/values/strings.xml]: This is wrong because it’s in the middle of a sentence.

File: res/values/strings.xml [hunk 2]
```diff
@@ -331,14 +331,14 @@
     <string name="move_file_invalid_into_descendent">It is not possible to move a folder into a descendant</string>
     <string name="move_file_invalid_overwrite">The file exists already in the destination folder</string>
     <string name="move_file_error">An error occurred while trying to move this file or folder</string>
-    <string name="forbidden_permissions_move">to move this file</string>
+    <string name="forbidden_permissions_move">To move this file</string>
```
Comment (jancborchardt) [inline res/values/strings.xml]: Same here

File: res/values/strings.xml [hunk 3]
```diff
@@ -331,14 +331,14 @@
...
 
 
     <string name="copy_file_not_found">Unable to copy. Please check whether the file exists</string>
     <string name="copy_file_invalid_into_descendent">It is not possible to copy a folder into a descendant</string>
     <string name="copy_file_invalid_overwrite">The file exists already in the destination folder</string>
     <string name="copy_file_error">An error occurred while trying to copy this file or folder</string>
-    <string name="forbidden_permissions_copy">to copy this file</string>
+    <string name="forbidden_permissions_copy">To copy this file</string>
```
Comment (jancborchardt) [inline res/values/strings.xml]: Same here

File: res/values/strings.xml [hunk 4]
```diff
@@ -372,7 +372,7 @@
     <string name="upload_move_files">Move file</string>
 
     <string name="pref_behaviour_entries_keep_file">kept in original folder</string>
-    <string name="pref_behaviour_entries_move">moved to app folder</string>
+    <string name="pref_behaviour_entries_move">Moved to app folder</string>
```
Comment (jancborchardt) [inline res/values/strings.xml]: Where is this used in the UI? Why is the string above then not capitalized?

Comment (tobiasKaminsky) [inline res/values/strings.xml]: Settings -> Original file will be...
moved to app folder
kept in original folder

**Comment by rperezb** [comment]
@logaprakash thanks for your contribution, it would be great if you send to us the contributor agreement signed, otherwise, we may not accept the pr
Besides, please, have a look at @jancborchardt comments

**Comment by davivel** [comment]
@karlitschek , could you check if @logaprakash sent the contribution agreement? Thanks in advance.

**Comment by jancborchardt** [comment]
@logaprakash you can also just state that your contribution is MIT licensed via commenting this here. It’s a bit simpler maybe.

**Comment by rperezb** [comment]
@davivel i confirm that @logaprakash has sent the contribution agreement

**Comment by rperezb** [comment]
@DeepDiver1975 he is part of the @owncloud/developer  and @owncloud/core-developers  can you add him to the @owncloud/android-developers one?

**Comment by YtvwlD** [comment]
@rperezb ... and you've successfully mentioned everyone. (I don't think, you can undo this. All members of the teams you named will receive any changes on this issue.)

**Comment by kyrofa** [comment]
@YtvwlD it's alright, we can unsubscribe individually at least :) .

**Comment by skjnldsv** [comment]
X), I was wondering what the hell was going on with all those crazy notifications :)
Keep up the good work!!

**Comment by DeepDiver1975** [comment]
@logaprakash you have been added to the android team - please rebase your branch to retrigger the bot - THX

**Pull request closed without merging**